recipe/session/main.go

```diff
@@ -367,17 +343,12 @@ func RevokeMultipleSessions(sessionHandles []string) ([]string, error) {
 	return RevokeMultipleSessionsWithContext(sessionHandles, &map[string]interface{}{})
 }
 
-func UpdateSessionData(sessionHandle string, newSessionData map[string]interface{}) (bool, error) {
-	return UpdateSessionDataWithContext(sessionHandle, newSessionData, &map[string]interface{}{})
-}
-
-// Deprecated: use MergeIntoAccessTokenPayload instead
-func UpdateAccessTokenPayload(sessionHandle string, newAccessTokenPayload map[string]interface{}) (bool, error) {
-	return UpdateAccessTokenPayloadWithContext(sessionHandle, newAccessTokenPayload, &map[string]interface{}{})
+func UpdateSessionDataInDatabase(sessionHandle string, newSessionData map[string]interface{}) (bool, error) {
+	return UpdateSessionDataInDatabaseWithContext(sessionHandle, newSessionData, &map[string]interface{}{})
 }
 
-func CreateJWT(payload map[string]interface{}, validitySecondsPointer *uint64) (jwtmodels.CreateJWTResponse, error) {
-	return CreateJWTWithContext(payload, validitySecondsPointer, &map[string]interface{}{})
+func CreateJWT(payload map[string]interface{}, useStaticSigningKey *bool, validitySecondsPointer *uint64) (jwtmodels.CreateJWTResponse, error) {
```

Comment:
useStaticSigningKey should be after validitySecondsPointer arg

Reply:
Fixed in https://github.com/supertokens/supertokens-golang/pull/273/commits/4a6a9339ba9b17e9383ed7b2704c1a38f36b3b3b